### PR TITLE
Added types for all Gtk functions

### PR DESCRIPTION
**Pull request by MrinmoyHaloi**

### CODE REVIEW

**Comment by MrinmoyHaloi** [comment]
I have added types for all of the no deprecated functions. I didn't added types for deprecated functions

File: gi-stubs/repository/Gtk.pyi [hunk 1]
```diff
@@ -17,7 +17,7 @@ from gi.repository import GLib
 from gi.repository import GObject
 from gi.repository import Pango
 import cairo
-
+import Gtk
```
Comment (lovetox) [inline gi-stubs/repository/Gtk.pyi]: I think this is a mistake here?

Comment (MrinmoyHaloi) [reply gi-stubs/repository/Gtk.pyi]: removed it

**Comment by MrinmoyHaloi** [comment]
Should we support the deprecated functions ?

**Comment by MrinmoyHaloi** [comment]
Squash the commit before merging

**Comment by lovetox** [comment]
> Should we support the deprecated functions ?

I think so, deprecation only means something if you want to upgrade to Gtk4, if someone has an application that runs Gtk3 its perfectly fine to use these methods.

**Comment by MrinmoyHaloi** [comment]
Ok I will add those in a different commit @lovetox

**Pull request merged**